packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json

```diff
@@ -155,5 +155,7 @@
   "ovhManagerSupport_new_business_cloud": "Cloud",
   "ovhManagerSupport_new_business_1": "Un projet à étudier?",
   "ovhManagerSupport_new_business_2": "Merci de nous détailler vos besoins en nous envoyant un e-mail à l'adresse :",
-  "ovhManagerSupport_new_business_3": "Les demandes d'assistances reçues à cette adresse ne seront pas traitées."
+  "ovhManagerSupport_new_business_3": "Les demandes d'assistances reçues à cette adresse ne seront pas traitées.",
+  "ovhManagerSupport_new_serviceType_publiccloud_gateway": "Gateway",
```

Comment:
Question: Is this needs translation ?

Reply:
@sachinrameshn no but should be there in all the files and I believe we should not update other files on our own.